docs/Recipes.md

```diff
@@ -10,31 +10,35 @@ You will need to install `ts-loader` for Haul to work with TypeScript.
 
 This is a `webpack.haul.js` that works with TypeScript.
```

Comment:
Could you also replace `webpack.haul.js` with `haul.config.js`?

Reply:
Sure! It's done.
(On a side note, `yarn haul init` is still creating `webpack.haul.js`)

Reply:
Thanks for the chatch @3vilguy I have created the issue for it #441